### PR TITLE
Add API Docs for Orders API

### PR DESCRIPTION
**Pull request by poush**
Partially Solves #4264
Fixes: #4290 
Fixes: #4299

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `nextgen` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Didn't added dredd tests because I guess @niranjan94 will add paypal sandbox credentials first to pass tests and docs is required in Urgent.

### CODE REVIEW

File: app/api/__init__.py [hunk 1]
```diff
@@ -321,7 +321,8 @@
 # attendees
 api.route(AttendeeListPost, 'attendee_list_post', '/attendees')
 api.route(AttendeeList, 'attendee_list', '/events/<int:event_id>/attendees',
-          '/events/<event_identifier>/attendees', '/orders/<int:order_id>/tickets/<int:ticket_id>/attendees')
+          '/events/<event_identifier>/attendees', '/orders/<int:order_id>/attendees',
+          '/<int:ticket_id>/attendees')
```
Comment (enigmaeth) [inline app/api/__init__.py]: Shouldn't this be `/tickets/<int:....>/attendees`?

Comment (poush) [inline app/api/__init__.py]: Yeah! Thanks :)

**Comment by SaptakS** [comment]
@poush travis build is failing. Please check.

File: app/api/__init__.py [hunk 2]
```diff
@@ -397,7 +398,7 @@
 api.route(OrdersListPost, 'order_list_post', '/orders')
 api.route(OrdersList, 'orders_list', '/orders', '/events/<int:event_id>/orders',
           '/events/<event_identifier>/orders')
-api.route(OrderDetail, 'order_detail', '/orders/<identifier>')
+api.route(OrderDetail, 'order_detail', '/orders/<identifier>', '/orders/<int:id>')
```
Comment (shubham-padia) [inline app/api/__init__.py]: Orders should not be accessible by id. The id is auto incremented and it's unsafe to use it in critical data like orders. Try mentioning identifier in self_view_kwargs when defining the schema if you are having any errors.

**Comment by poush** [comment]
@fossasia/open-event-devs review please

File: app/api/__init__.py [hunk 3]
```diff
@@ -326,7 +326,8 @@
 # attendees
 api.route(AttendeeListPost, 'attendee_list_post', '/attendees')
 api.route(AttendeeList, 'attendee_list', '/events/<int:event_id>/attendees',
-          '/events/<event_identifier>/attendees', '/orders/<int:order_id>/tickets/<int:ticket_id>/attendees')
+          '/events/<event_identifier>/attendees', '/orders/<int:order_id>/attendees',
```
Comment (shubham-padia) [inline app/api/__init__.py]: I think the above `<int:order_id>` was missed in changing to order_identifier, can you confirm please.
Rest looks good.

File: app/api/__init__.py [hunk 4]
```diff
@@ -336,7 +337,7 @@
 api.route(EventTypeList, 'event_type_list', '/event-types')
 api.route(EventTypeDetail, 'event_type_detail', '/event-types/<int:id>', '/events/<int:event_id>/event-type',
           '/events/<event_identifier>/event-type')
-api.route(EventTypeRelationship, 'event_type_event', '/event-types/<int:id>/relationships/events')
+api.route(EventTypeRelationship, 'event_type_event', '/event-types/<int:id>/relationships/1events')
```
Comment (SaptakS) [inline app/api/__init__.py]: I think this is a mistake. Please check.

File: tests/hook_main.py [hunk 5]
```diff
@@ -3042,18 +3041,15 @@ def event_export_post(transaction):
 
 
 # ------------------------- Import -------------------------
-@hooks.before(
-    "Event Import > Start Event Import > Start a Task to Import an Event")
+@hooks.before("Event Import > Start Event Import > Start a Task to Import an Event")
```
Comment (SaptakS) [inline tests/hook_main.py]: What exactly does this line improve?

**Pull request merged**